### PR TITLE
Update retl bq docs

### PR DESCRIPTION
**Pull request by gilsegment**
## Ticket 
https://segment.atlassian.net/browse/RETL-1617

## Proposed changes
Updating Reverse-ETL BigQuery documentation. It seems that it wasn't fully up to date. 

### Merge timing
ASAP

## Related issues (optional)
[Box BigQuery permission issue](https://segment.atlassian.net/browse/RETL-1558)

### CODE REVIEW

**Review by prayansh — APPROVED** [review]
couple suggestions. but looks good overall!

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 1]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
...
-`bigquery.datasets.get` | This allows Segment to determine if the aforementioned dataset exists.
-`bigquery.jobs.create` | This allows Segment to execute queries on any datasets or tables your model query references, and also allows Segment to manage tables used for tracking.
+### Grant Full Access
+With this approach we will use BigQuery predefined roles: 
+1. Navigate to **IAM & Admin > Service Accounts** in BigQuery.
+2. Click **+ Create Service Account** to create a new service account.
+3. Enter your **Service account name** and a description of what the service-account will do.
+4. Click **Create and Continue**. 
+5. Click **+ Add another role** and add the **BigQuery User** role. 
+6. Click **+ Add another role** and add the **BigQuery Data Editor** role.
```
Comment (prayansh) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:24]:
```suggestion
6. Click **+ Add another role** and add the [**BigQuery Data Editor**](https://cloud.google.com/bigquery/docs/access-control#bigquery.dataEditor) role. 
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 2]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
...
-`bigquery.datasets.create` | This allows Segment to create/manage a `__segment_reverse_etl` dataset for tracking state between syncs.
-`bigquery.datasets.get` | This allows Segment to determine if the aforementioned dataset exists.
-`bigquery.jobs.create` | This allows Segment to execute queries on any datasets or tables your model query references, and also allows Segment to manage tables used for tracking.
+### Grant Full Access
+With this approach we will use BigQuery predefined roles: 
+1. Navigate to **IAM & Admin > Service Accounts** in BigQuery.
+2. Click **+ Create Service Account** to create a new service account.
+3. Enter your **Service account name** and a description of what the service-account will do.
+4. Click **Create and Continue**. 
+5. Click **+ Add another role** and add the **BigQuery User** role.
```
Comment (prayansh) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:23]:
```suggestion
5. Click **+ Add another role** and add the [**BigQuery User**](https://cloud.google.com/bigquery/docs/access-control#bigquery.user) role. 
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 3]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
...
+8. Enter your **Service account name** and a description of what the account will do. 
+9. Click **Create and Continue**. 
+10. In the **Grant this service account access to project** section, select the role you just created. 
+11. Click **Continue**. 
+12. Click **Done**. 
+13. Navigate to the BigQuery SQL editor and create a dataset that will be used by Segment:
+    ```
+    CREATE SCHEMA IF NOT EXISTS `__segment_reverse_etl`;
+    ```
+14. Grant additional permissions just on the newly created dataset:
```
[the remark this answers is not in the document]
Comment (prayansh) [reply src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:53]: would be good to somehow link this URL to the BQ docs for that role, https://cloud.google.com/bigquery/docs/access-control#bigquery.dataEditor.

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 4]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
 redirect_from:
   - '/reverse-etl/bigquery-setup/'
 ---
-
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:6]: hi @gilsegment - the docs team typically adds an intro sentence to help signpost for customers who may be scanning where they are and what they'll accomplish on a page - feel free to adjust this sentence as needed!

```suggestion

To set up your BigQuery source with Reverse ETL, you must [construct a BigQuery role and service account](#constructing-your-own-role-or-policy] and [create a BigQuery source in the Segment app](#set-up-bigquery-as-your-reverse-etl-source).

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 5]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
...
-6. Click **Add source** if the test connection is successful.
-
-After you've added BigQuery as a source, you can [add a model](/docs/connections/reverse-etl/setup/#step-2-add-a-model).
-
-> info "BigQuery Reverse ETL sources support Segment's dbt extension"
-> If you have an existing dbt account with a Git repository, you can use [Segment's dbt extension](/docs/segment-app/extensions/dbt/) to centralize model management and versioning, reduce redundancies, and run CI checks to prevent breaking changes.
+When creating a role and service-account, we offer two approaches:
+1. **Grant Full Access**: This option provides Segment with all the necessary permissions and allows us to complete the setup for you automatically. It's the quicker option, requiring minimal effort on your part.
+2. **Grant Limited Access**: This option is more secure, as it restricts permissions. However, due to the limited access, a few additional setup steps will need to be completed manually by you. These are one-time steps, and we will guide you through the process.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:13]:
```suggestion
There are two approaches you can take when granting Segment access to your BigQuery resources: 
1. **Grant Full Access**: This option allows Segment to automatically complete the setup for you after you provide Segment with all the necessary permissions. This option requires less time and engineering effort on your part.
2. **Grant Limited Access**: This option is more secure, as it restricts the permissions Segment has access to. However, due to the limited access, you must complete a few additional setup steps. These are one-time steps, and the documentation provides you with the information required to complete this process.
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 6]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
...
 
-## Constructing your own role or policy
-When you construct your own role or policy, Segment needs the following permissions:
+You are free to choose the approach that best suits your needs, and can skip the other.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:15]:
```suggestion
You can choose the approach that best suits your needs.
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 7]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
...
 
-Permission | Details
----------- | --------
-`bigquery.datasets.create` | This allows Segment to create/manage a `__segment_reverse_etl` dataset for tracking state between syncs.
-`bigquery.datasets.get` | This allows Segment to determine if the aforementioned dataset exists.
-`bigquery.jobs.create` | This allows Segment to execute queries on any datasets or tables your model query references, and also allows Segment to manage tables used for tracking.
+### Grant Full Access
+With this approach we will use BigQuery predefined roles: 
+1. Navigate to **IAM & Admin > Service Accounts** in BigQuery.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:19]: The docs team typically includes the program that you're in both for the first steps and for any steps where you change from one program to another!

```suggestion
### Grant Full Access
With this approach, use BigQuery predefined roles to create a service account for Segment to assume. 
1. In BigQuery, navigate to **IAM & Admin > Service Accounts**.
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 8]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
...
+2. Click **+ Create Service Account** to create a new service account.
+3. Enter your **Service account name** and a description of what the service-account will do.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:21]:
```suggestion
3. Enter your **Service account name** and a description of what the service account will do.

```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 9]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
...
+4. Click **Create and Continue**. 
+5. Click **+ Add another role** and add the **BigQuery User** role. 
+6. Click **+ Add another role** and add the **BigQuery Data Editor** role. 
+7. Click **Continue**. 
+8. Click **Done**.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:26]:
```suggestion
7. Click **Continue**., then click **Done**.
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 10]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
...
 
-If you don't want to grant Segment `bigquery.datasets.create` access, you can create your own `__segment_reverse_etl` dataset and give Segment the following permissions: 
+### Grant Limited Access
+With this approach we will use custom role with the below permissions:
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:29]:
```suggestion
With this approach, create a custom role for Segment with the following permissions:
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 11]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
...
 ---------- | --------
 `bigquery.datasets.get` | This allows Segment to determine if the `__segment_reverse_etl` dataset exists.
-`bigquery.tables.create` | This allows Segment to determine if the tables Segment uses to track state in the `__segment_reverse_etl` dataset exists.
 `bigquery.jobs.create` | This allows Segment to execute queries on any datasets or tables your model query references, and also allows Segment to manage tables used for tracking.
 
-The `bigquery.datasets.*` permissions can be scoped only to the `__segment_reverse_etl` dataset. 
 
-After you've successfully added your BigQuery source, [add a model](/docs/connections/reverse-etl/setup/#step-2-add-a-model) and follow the rest of the steps in the Reverse ETL setup guide. 
\ No newline at end of file
+1. Navigate to **IAM & Admin > Roles** in BigQuery.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:37]: Same as the note above - starting with the program here

```suggestion
1. In BigQuery, navigate to **IAM & Admin > Roles**.
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 12]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
...
+2. Click **+ CREATE ROLE** to create a new role.
+3. Add **title** and **Description** as you like.
+4. Click **ADD PERMISSIONS** and add the permission listed in the above tables. You will to repeat that until all required permissions were added.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:40]:
```suggestion
3. Add a **Title** for your role and a **Description** of what the role will have access to.
4. Click **ADD PERMISSIONS** and add all of the permissions listed in the table above.
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 13]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
...
+5. Click **CREATE**. 
+6. Navigate to **IAM & Admin > Service Accounts** in BigQuery.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:42]: When we stay in the same program, we typically don't note which program we're in!

```suggestion
6. Navigate to **IAM & Admin > Service Accounts**. 
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 14]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
...
+3. Add **title** and **Description** as you like.
+4. Click **ADD PERMISSIONS** and add the permission listed in the above tables. You will to repeat that until all required permissions were added.
+5. Click **CREATE**. 
+6. Navigate to **IAM & Admin > Service Accounts** in BigQuery. 
+7. Click **+ Create Service Account** to create a new service account. 
+8. Enter your **Service account name** and a description of what the account will do. 
+9. Click **Create and Continue**. 
+10. In the **Grant this service account access to project** section, select the role you just created. 
+11. Click **Continue**. 
+12. Click **Done**.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:48]: The docs site automatically has the correct numbering, so no need to worry about the rest of the items in this list!

```suggestion
11. Click **Continue**, then click **Done**. 
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 15]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
...
+5. Click **CREATE**. 
+6. Navigate to **IAM & Admin > Service Accounts** in BigQuery. 
+7. Click **+ Create Service Account** to create a new service account. 
+8. Enter your **Service account name** and a description of what the account will do. 
+9. Click **Create and Continue**. 
+10. In the **Grant this service account access to project** section, select the role you just created. 
+11. Click **Continue**. 
+12. Click **Done**. 
+13. Navigate to the BigQuery SQL editor and create a dataset that will be used by Segment:
+    ```
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:50]: This adds SQL syntax highlighting!

```suggestion
    ```sql
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 16]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
...
+    CREATE SCHEMA IF NOT EXISTS `__segment_reverse_etl`;
+    ```
+14. Grant additional permissions just on the newly created dataset:
+    ```
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:54]:
```suggestion
    ```sql
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 17]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
...
+    GRANT `roles/bigquery.dataEditor` ON SCHEMA `__segment_reverse_etl` TO "serviceAccount:<YOUR SERVICE ACCOUNT EMAIL>";
+    ```
+
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:57]: I know this was its own few steps in the procedure for creating a BigQuery source in Segment, but it's probably a prerequisite and needs its own mini-procedure! 

```suggestion

### BigQuery resource location
When connecting your BigQuery warehouse to Segment, you'll need to know the location of your resources.

You can find the location of your BigQuery resources using the following method:
1. In the BigQuery console, navigate to your dataset. In the explorer panel on the left, expand the project and dataset to view the tables.
2. Click on the name of the dataset, and it opens a page showing its details.
3. The Location of the dataset (like US or EU) is displayed in the Dataset Info.

```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 18]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
...
+## Set up BigQuery as your Reverse ETL source
+1. In the BigQuery console, search for the service account you just created.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:59]:
```suggestion
1. In the BigQuery console, search for the service account you created. 
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 19]
```diff
@@ -3,53 +3,73 @@ title: BigQuery Reverse ETL Setup
...
+3. Click **Add Key > Create new key**. 
+4. In the pop-up window, select **JSON** for the key type and click **Create**. The file will be downloaded. 
+5. Copy all the content within the **JSON** file created and downloaded in previous section. 
+6. Navigate to the Segment UI and paste all the credentials you copied from previous step into the **Enter your credentials** section. 
+7. Enter your **Data Location**. The data location can be found by:
+   - In the BigQuery console navigate to your dataset: In the explorer panel on the left, expand the project and dataset to view the tables.
+   - Click on the name of the dataset, and it will open a page showing its details.
+   - The Location of the dataset (like US or EU) is displayed in the Dataset Info.
+8. Click **Test Connection** to test to see if the connection works. If the connection fails, make sure you have the right permissions and credentials and try again. 
+9. Click **Add source** if the test connection is successful.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:70]:
```suggestion
5. Copy all the content in the JSON file you created in the previous step. 
6. Open the Segment app and navigate to **Connections > Sources**. 
7. On the My sources page, click **+ Add source**.  
8. Search for "BigQuery" and select the BigQuery source from the sources catalog. On the BigQuery overview page, click **Add Source**. 
9. On the Set up BigQuery page, enter a name for your source and paste all the credentials you copied from previous step into the **Enter your credentials** section. 
10. Enter the location of your BigQuery warehouse in the **Data Location** field. 
11. Click **Test Connection** to test to see if the connection works. If the connection fails, make sure you have the right permissions and credentials and try again. 
12. If the test connection completes successfully, click **Add source** to complete the setup process.
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 20]
```diff
@@ -3,53 +3,82 @@ title: BigQuery Reverse ETL Setup
 redirect_from:
   - '/reverse-etl/bigquery-setup/'
 ---
+> info "BigQuery Reverse ETL sources support Segment's dbt extension"
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:6]: There needs to be an introductory sentence here - feel free to edit this comment to make changes to this sentence if you think something else would work better.

```suggestion
To set up your BigQuery source with Reverse ETL, you must [construct a BigQuery role and service account](#constructing-your-own-role-or-policy] and [create a BigQuery source in the Segment app](#set-up-bigquery-as-your-reverse-etl-source). 

> info "BigQuery Reverse ETL sources support Segment's dbt extension"
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 21]
```diff
@@ -3,53 +3,82 @@ title: BigQuery Reverse ETL Setup
...
-14. Navigate to the Segment UI and paste all the credentials you copied from step 13 into the **Enter your credentials** section.
-19. Enter your **Data Location**.
-20. Click **Test Connection** to test to see if the connection works. If the connection fails, make sure you have the right permissions and credentials and try again.
-6. Click **Add source** if the test connection is successful.
-
-After you've added BigQuery as a source, you can [add a model](/docs/connections/reverse-etl/setup/#step-2-add-a-model).
+There are two approaches you can take when granting Segment access to your BigQuery resources: 
+1. **Grant Full Access**: This option allows Segment to automatically complete the setup for you after you provide Segment with all the necessary permissions. This option requires less time and engineering effort on your part.
+2. **Grant Limited Access**: This option is more secure, as it restricts the permissions Segment has access to. However, due to the limited access, you must complete a few additional setup steps. These are one-time steps, and the documentation provides you with the information required to complete this process.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:16]: This should be an unordered list unless these options are ranked in order or are sequential steps

```suggestion
- **Grant Full Access**: This option allows Segment to automatically complete the setup for you after you provide Segment with all the necessary permissions. This option requires less time and engineering effort on your part.
- **Grant Limited Access**: This option is more secure, as it restricts the permissions Segment has access to. However, due to the limited access, you must complete a few additional setup steps. These are one-time steps, and the documentation provides you with the information required to complete this process.
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 22]
```diff
@@ -3,53 +3,82 @@ title: BigQuery Reverse ETL Setup
...
+3. Enter your **Service account name** and a description of what the service account will do.
 
-Permission | Details
----------- | --------
-`bigquery.datasets.create` | This allows Segment to create/manage a `__segment_reverse_etl` dataset for tracking state between syncs.
-`bigquery.datasets.get` | This allows Segment to determine if the aforementioned dataset exists.
-`bigquery.jobs.create` | This allows Segment to execute queries on any datasets or tables your model query references, and also allows Segment to manage tables used for tracking.
+4. Click **Create and Continue**. 
+5. Click **+ Add another role** and add the [**BigQuery User**](https://cloud.google.com/bigquery/docs/access-control#bigquery.user) role. 
+6. Click **+ Add another role** and add the [**BigQuery Data Editor**](https://cloud.google.com/bigquery/docs/access-control#bigquery.dataEditor) role.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:28]: All links that don't point to a resource on the segment.com/docs subdomain need the `{:target="_blank”}` attribute.

```suggestion
5. Click **+ Add another role** and add the [**BigQuery User**](https://cloud.google.com/bigquery/docs/access-control#bigquery.user){:target="_blank”} role. 
6. Click **+ Add another role** and add the [**BigQuery Data Editor**](https://cloud.google.com/bigquery/docs/access-control#bigquery.dataEditor){:target="_blank”} role. 
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 23]
```diff
@@ -3,53 +3,82 @@ title: BigQuery Reverse ETL Setup
...
-`bigquery.datasets.get` | This allows Segment to determine if the aforementioned dataset exists.
-`bigquery.jobs.create` | This allows Segment to execute queries on any datasets or tables your model query references, and also allows Segment to manage tables used for tracking.
+4. Click **Create and Continue**. 
+5. Click **+ Add another role** and add the [**BigQuery User**](https://cloud.google.com/bigquery/docs/access-control#bigquery.user) role. 
+6. Click **+ Add another role** and add the [**BigQuery Data Editor**](https://cloud.google.com/bigquery/docs/access-control#bigquery.dataEditor) role. 
+7. Click **Continue**, then click **Done**. 
 
-If you don't want to grant Segment `bigquery.datasets.create` access, you can create your own `__segment_reverse_etl` dataset and give Segment the following permissions: 
+### Grant Limited Access
+With this approach we will setup a custom role with the following permissions:
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:32]: "setup" is a noun, "set up" is a verb.

```suggestion
With this approach, you can set up a custom role with the following permissions:
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 24]
```diff
@@ -3,53 +3,82 @@ title: BigQuery Reverse ETL Setup
...
 `bigquery.datasets.get` | This allows Segment to determine if the `__segment_reverse_etl` dataset exists.
-`bigquery.tables.create` | This allows Segment to determine if the tables Segment uses to track state in the `__segment_reverse_etl` dataset exists.
 `bigquery.jobs.create` | This allows Segment to execute queries on any datasets or tables your model query references, and also allows Segment to manage tables used for tracking.
+`bigquery.tables.getData` | This allows Segment to run `SELECT` queries on tables that will be defined in the model. 
+
+
+1. In BigQuery, navigate to **IAM & Admin > Roles**.
+2. Click **+ CREATE ROLE** to create a new role.
+3. Add **Title** and **Description** as you like.
+4. Click **ADD PERMISSIONS** and add the permission listed in the above tables. You will need to repeat that until all required permissions are added.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:44]: The second sentence should be in active voice!

```suggestion
4. Click **ADD PERMISSIONS** and add the permission listed in the above tables. Repeat this step until you've added all required permissions.
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 25]
```diff
@@ -3,53 +3,82 @@ title: BigQuery Reverse ETL Setup
...
 
-The `bigquery.datasets.*` permissions can be scoped only to the `__segment_reverse_etl` dataset. 
+## Set up BigQuery as your Reverse ETL source
+1. In the BigQuery console, search for the service account you created. 
+2. When your service account pulls up, click the 3 dots under **Actions** and select **Manage keys**. 
+3. Click **Add Key > Create new key**. 
+4. In the pop-up window, select **JSON** for the key type and click **Create**. The file will be downloaded. 
+5. Copy all the content in the JSON file you created in the previous step. 
+6. Open the Segment app and navigate to **Connections > Sources**. 
+7. On the _My sources_ page, click **+ Add source**.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:77]:
```suggestion
7. On the My sources page, click **+ Add source**.  
```

File: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md [hunk 26]
```diff
@@ -3,53 +3,82 @@ title: BigQuery Reverse ETL Setup
...
+## Set up BigQuery as your Reverse ETL source
+1. In the BigQuery console, search for the service account you created. 
+2. When your service account pulls up, click the 3 dots under **Actions** and select **Manage keys**. 
+3. Click **Add Key > Create new key**. 
+4. In the pop-up window, select **JSON** for the key type and click **Create**. The file will be downloaded. 
+5. Copy all the content in the JSON file you created in the previous step. 
+6. Open the Segment app and navigate to **Connections > Sources**. 
+7. On the _My sources_ page, click **+ Add source**.  
+8. Search for "BigQuery" and select the BigQuery source from the sources catalog. On the BigQuery overview page, click **Add Source**. 
+9. On the _Set up BigQuery_ page, enter a name for your source and paste all the credentials you copied from previous step into the **Enter your credentials** section.
```
Comment (forstisabella) [inline src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md:79]:
```suggestion
9. On the Set up BigQuery page, enter a name for your source and paste all the credentials you copied from previous step into the **Enter your credentials** section. 
```

**Review by forstisabella — COMMENTED** [review]
a few changes and we'll be g2g!

**Review by forstisabella — APPROVED** [review]

**Pull request merged**